tests/rbd/test_rbd_deep_flatten_negative.py

```diff
@@ -9,14 +9,71 @@
 log = Log(__name__)
 
 
+def verify_deep_flatten(clone_format, rbd, pool, image, snap_spec):
+    """ """
```

Comment:
why empty doc string?

Reply:
that was how it was when that test was automated. Changing that is not part of this PR. I can do it separately in a different PR.